src/test/java/com/epam/jamp2/service/impl/CalculationCommandExecutionServiceTest.java

```diff
@@ -0,0 +1,158 @@
+                        FixerioResponse response = new FixerioResponse();
+                        String currency = (String) invocationOnMock.getArguments()[0];
+                        response.setRates(testRates.get(currency.toUpperCase()));
+                        return Response.success(response);
+                    }
+
+                    @Override
+                    public void enqueue(Callback<FixerioResponse> callback) {
+                    }
+
+                    @Override
+                    public boolean isExecuted() {
+                        return false;
+                    }
+
+                    @Override
+                    public void cancel() {
+                    }
+
+                    @Override
+                    public boolean isCanceled() {
+                        return false;
+                    }
+
+                    @Override
+                    public Call<FixerioResponse> clone() {
+                        return null;
+                    }
+
+                    @Override
+                    public Request request() {
+                        return null;
+                    }
+                });
+
+        //Inject mocked service to related class
+        calculationCommandExecutionService = new CalculationCommandExecutionServiceImpl();
+        calculationCommandExecutionService.fxRatesService = fxRatesServiceImpl;
+    }
+
+    @Test
+    public void testConversionUnnecessary() throws IOException, UnknownCurrencyException {
+        CalculationCommand calculationCommand = new CalculationCommand(
+                new Value("usd", BigDecimal.valueOf(100)),
+                new Value("usd", BigDecimal.valueOf(5)),
+                Operation.ADD, "usd");
+        assertThat(calculationCommandExecutionService.calculate(calculationCommand).getValue(),
+                greaterThan(BigDecimal.ZERO));
+        calculationCommand = new CalculationCommand(
+                new Value("cny", BigDecimal.valueOf(100)),
+                new Value("cny", BigDecimal.valueOf(5)),
+                Operation.SUBTRACT, "cny");
+        assertThat(calculationCommandExecutionService.calculate(calculationCommand).getValue(),
+                greaterThan(BigDecimal.ZERO));
+        calculationCommand = new CalculationCommand(
+                new Value("hkd", BigDecimal.valueOf(100)),
+                new Value("hkd", BigDecimal.valueOf(5)),
+                Operation.MULTIPLY, "hkd");
+        assertThat(calculationCommandExecutionService.calculate(calculationCommand).getValue(),
+                greaterThan(BigDecimal.ZERO));
+        calculationCommand = new CalculationCommand(
+                new Value("jpy", BigDecimal.valueOf(100)),
+                new Value("jpy", BigDecimal.valueOf(5)),
+                Operation.DIVIDE, "jpy");
+        assertThat(calculationCommandExecutionService.calculate(calculationCommand).getValue(),
```

Comment:
Asserts() that you do in this test are not really related to what is implied by the name of the test "testConversionUnnecessary()". So I would recommend to extract them to separate test. 

Also asserting that result is simply greaterThan(zero) is quite broad in nature, I would recommend you to make your checks more specific, e.g. assert that 100 ADD 5 is 105. 

Another thing is testing different mathematics operations might be considered as redundant, you can alway say that 10 + 5 = 10 - (-5); 10*2 = 10+10, etc. Exception for the case 10/0